lib/controllers/services.ts

```diff
@@ -184,11 +213,68 @@ export function GetServicesByRecipient(
     AzureApiAuthMiddleware(new Set([UserGroup.ApiServiceByRecipientQuery])),
     ClientIpMiddleware,
     azureUserAttributesMiddleware,
-    RequiredQueryParamMiddleware("recipient", FiscalCode)
+    RequiredParamMiddleware("fiscalcode", FiscalCode)
   );
   return wrapRequestHandler(
     middlewaresWrap(
       checkSourceIpForHandler(handler, (_, c, u, __) => ipTuple(c, u))
     )
   );
 }
+
+///////////////////////////////////////
+
+/**
+ * Returns the serviceId for all the Services that have sent
+ * at least one notification to the recipient with the provided fiscalCode.
+ */
+export function GetVisibleServicesHandler(
+  blobService: BlobService
+): IGetVisibleServicesHandler {
+  return async (_, __, ___) => {
+    const errorOrVisibleServicesJson = await getBlobAsObject(
```

Comment:
can't we use the input binding here?

Reply:
I fear that if we use input bindings here (in public_api_v1.ts -> index(.., , myblob)))
we end up to have an extra request to the blob storage even when the API HTTP call is directed towards another controller. What do you think about it ?

Reply:
that's a very good point and I think you are right as the function is just one for all controllers 👍